modules/runner/src/integrationTest/java/org/apache/ignite/internal/ssl/ItSslTest.java

```diff
@@ -305,5 +338,23 @@ void clientCanConnectWithSslAndClientAuth() throws Exception {
                 assertThat(client.clusterNodes(), hasSize(2));
             }
         }
+
+        @Test
+        @DisplayName("Jdbc client can connect with SSL configured")
+        void jdbcCanConnectWithSslAndClientAuth() throws SQLException {
+            var url =
+                    "jdbc:ignite:thin://127.0.0.1:10800"
+                            + "?sslEnabled=true"
+                            + "&trustStorePath=" + trustStorePath
+                            + "&trustStoreType=JKS"
+                            + "&trustStorePassword=" + password
+                            + "&clientAuth=require"
```

Comment:
Do we support clientAuth=optional\none ?

Reply:
This is supported by Netty SSL handler implementation. So, basically yes, it is supported.

Reply:
But probably, the integration test is not the best place to setup all possible configurations for the cluster.

Reply:
Ok, we check only connectivity here.

I think we can create a ticket to add more tests later, to check supported cyphers and clientAuth option, and maybe smth else.